pyproject.toml

```diff
@@ -19,20 +19,18 @@ documentation = "https://masterblasterpy.readthedocs.io/en/latest/"
 
 [tool.poetry.dependencies]
 python = "^3.10"
-aiohttp = "^3.8.5"
-python-dateutil = "^2.8.2"
-
-[tool.poetry.dev-dependencies]
+aiohttp = "^3.10.1"
+python-dateutil = "^2.9.0.post0"
```

Comment:
This is the intended version, right? 🤔

Reply:
Apparently!
Haven't seen that naming before, but it is what pypi says 🤷